packages/components/ovh-payment-method/src/ovh-payment-mean.js

```diff
@@ -0,0 +1,294 @@
+  const usedReketInstance = reketInstance;
+  const usedRegion = region;
+
+  const buildRejectError = (errorObject) => {
+    const paymentMeanError = new Error(errorObject.data.message);
+    paymentMeanError.status = errorObject.status;
+    paymentMeanError.data = errorObject.data;
+    return paymentMeanError;
+  };
+
+  /*= ==============================================
+  =            Available Payment Means            =
+  =============================================== */
+
+  const getAvailablePaymentMeans = (
+    options = DEFAULT_GET_AVAILABLE_OPTIONS,
+  ) => {
+    return usedReketInstance
+      .get('/me/availableAutomaticPaymentMeans')
+      .then((availableAutomaticPaymentMeans) => {
+        const availablePaymentMeansInfos =
+          AVAILABLE_PAYMENT_MEAN_TYPES[usedRegion] || [];
+
+        return availablePaymentMeansInfos
+          .filter(({ value, registerable }) => {
+            if (!availableAutomaticPaymentMeans[value]) {
+              return false;
+            }
+
+            if (options.onlyRegisterable && !registerable) {
+              return false;
+            }
+
+            return true;
+          })
+          .map(({ value, registerable }) => {
+            const availablePaymentMean = new AvailablePaymentMean({
+              value,
+              registerable,
+            });
+
+            return options.transform
+              ? availablePaymentMean.toAvailablePaymentMethod()
+              : availablePaymentMean;
+          });
+      });
+  };
+
+  /* -----  End of Available Payment Means  ------*/
+
+  /*= ===============================================
+  =            Actions on payment means            =
+  ================================================ */
+
+  const addPaymentMean = (availablePaymentMean, params = {}) => {
+    if (usedRegion === 'US') {
+      return Promise.reject(
+        buildRejectError({
+          status: 403,
+          data: {
+            message: 'addPaymentMean is not available for US world part',
+          },
+        }),
+      );
+    }
+
+    const addParams = params;
+
+    if (has(addParams, 'default')) {
+      addParams.setDefault = addParams.default;
+      delete addParams.default;
+    }
+
+    return usedReketInstance
+      .post(`/me/paymentMean/${availablePaymentMean.meanType}`, addParams)
+      .then((result) => {
+        if (
+          result.url &&
+          availablePaymentMean.meanType !== PAYMENT_MEAN_TYPE_ENUM.bankAccount
+        ) {
+          if (!params.returnUrl) {
+            window.open(result.url, '_blank');
+          } else {
+            window.location = result.url;
+          }
+        }
+
+        return result;
+      });
+  };
+
+  /**
+   *  Edit the given payment mean.
+   *
+   *  @param  {PaymentMean} paymentMean The payment mean to edit
+   *  @return {Promise}
+   */
+  const editPaymentMean = (paymentMean, params) => {
+    if (usedRegion === 'US') {
```

Comment:
US is used in multiple places. Do you want to extract it as a constant?

Reply:
Good point :-) I won't change this now as payment mean will be soon deprecated. This means that right after the first release of ovh-payment-method, all methods regarding payment means will be deprecated too and then removed.